pkg/util/flag/tristate.go

```diff
@@ -14,37 +14,37 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package util
+package flag
 
 import (
 	"fmt"
 	"strconv"
 )
 
-// BoolFlag is a boolean flag compatible with flags and pflags that keeps track of whether it had a value supplied or not.
+// Tristate is a boolean flag compatible with flags and pflags that keeps track of whether it had a value supplied or not.
 // Getting this flag to act like a normal bool, where true/false are not required needs a little bit of extra code, example:
-//    f := cmd.Flags().VarPF(&BoolFlagVar, "flagname", "", "help about the flag")
+//    f := cmd.Flags().VarPF(&TristateVar, "flagname", "", "help about the flag")
 //    f.NoOptDefVal = "true"
-type BoolFlag struct {
+type Tristate struct {
```

Comment:
I'd recommend making this e.g.

```go
type Tristate int

const (
  Unset Tristate = iota
  True
  False
)
```

Because it's a bit weird that right now there's four possible configurations...